requirements.txt

```diff
@@ -11,6 +11,7 @@ dfdatetime >= 20221112
 dfvfs >= 20221224
 dfwinreg >= 20211207
 dtfabric >= 20220219
+flor
```

Comment:
this will need to be added to dependencies.ini

Reply:
Looks like Ubuntu has the 1.1.3 version - https://packages.ubuntu.com/jammy/python3-flor

No results for Fedora

Reply:
* done (c418b5935f940421f873f66a3c1aff7ebdf54276)
* need manual modification to reflect change made in e93da822c5f567a7a1a2f869ef3b83eed3fdeaf4 to [config/dpkg/control](https://github.com/log2timeline/plaso/commit/e93da822c5f567a7a1a2f869ef3b83eed3fdeaf4#diff-5e44eafdee7bfb04feaa317e5c1497264895439cd10d887e256bebf642655c95)

Reply:
done ( c418b5935f940421f873f66a3c1aff7ebdf54276 )
    need manual modification to reflect change made in e93da822c5f567a7a1a2f869ef3b83eed3fdeaf4 to config/dpkg/control

Reply:
* done in [c418b59](https://github.com/log2timeline/plaso/pull/4527/commits/c418b5935f940421f873f66a3c1aff7ebdf54276)
* Manual modification to reflect change on [config/dpkg/control](https://github.com/log2timeline/plaso/commit/e93da822c5f567a7a1a2f869ef3b83eed3fdeaf4#diff-5e44eafdee7bfb04feaa317e5c1497264895439cd10d887e256bebf642655c95)  in e93da822c5f567a7a1a2f869ef3b83eed3fdeaf4